srcpkgs/gnome-desktop/template

```diff
@@ -26,8 +26,8 @@ case "$XBPS_TARGET_MACHINE" in
 esac
 
 gnome-desktop-devel_package() {
-	depends="libxkbfile-devel gsettings-desktop-schemas-devel>=3.12
-		gtk+3-devel ${sourcepkg}>=${version}_${revision}"
+	depends="libseccomp-devel libxkbfile-devel gtk+3-devel
+ gsettings-desktop-schemas-devel>=3.12 ${sourcepkg}>=${version}_${revision}"
```

Comment:
sorry, expressed myself poorly.

1 whitespace indent starting from the same position as depends=

Reply:
oops, missed a tab